go.mod

```diff
@@ -263,3 +263,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	stash.kopano.io/kgol/kcc-go/v5 v5.0.1 // indirect
 )
+
+replace github.com/cs3org/go-cs3apis => github.com/kobergj/go-cs3apis v0.0.0-20220610101249-8067918fe8bf
```

Comment:
I'd love to. I have the PR here: https://github.com/cs3org/cs3apis/pull/172 it just needs approve&merge

Reply:
AFAICT this PR does not use the grant Creator... is this line even necessary for this PR? If so can you point me to the place?

Reply:
It wail break the build without the line. It needs the updated cs3apis when using the current reva. I assume that is because go-cs3apis is not an indirect dependency. It is used in several other places. Using different version of it for ocis and reva will confuse the compiler